api/src/main/java/org/infinispan/api/sync/MultiMap.java

```diff
@@ -0,0 +1,32 @@
+package org.infinispan.api.sync;
+
+import org.infinispan.api.configuration.MultiMapConfiguration;
+
+/**
+ *
+ * @param <K>
+ * @param <V>
+ * @since 13.0
+ */
+public interface MultiMap<K, V> {
+
+   String name();
+
+   MultiMapConfiguration configuration();
+
+   Void add(K key, V value);
+
+   Iterable<V> get(K key);
+
+   boolean remove(K key);
+
+   boolean remove(K key, V value);
+
+   boolean containsKey(K key);
+
+   boolean containsValue(V value);
```

Comment:
Do we want to have this method exposed? And if we do should it not be `long containedCount(V value)` or something like that?

Reply:
This all depends on how users want to use this.